go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -432,13 +432,26 @@ func (c *Controller) addServiceToIdlingBalancer(vips sets.String, service *v1.Se
 	return nil
 }
 
+// RequestFullSync re-syncs every service that currently exists
+func (c *Controller) RequestFullSync() error {
+	klog.Info("Full service sync requested")
+	services, err := c.serviceLister.List(labels.Everything())
+	if err != nil {
+		return err
+	}
+	for _, service := range services {
```

Comment:
all services because we have to include any that may be moved to a node specific load balancer. These include:
1. nodeport
2. services with host network pods
3. services with external traffic policy local (TBD when Andrew's PR gets in)

Reply:
this is a performance handicap ... a big one but :shrug: